cmd/logon.go

```diff
@@ -53,6 +59,10 @@ var logonCmd = &cobra.Command{
 			password = string(byteSecretVal)
 		}
 
+		if Password != "" {
+			password = Password
```

Comment:
The `cybr ccp get-account` command returns the full CCP JSON response blob and doesn't parse the password out to allow command substitution.  We should either provide a flag in `cybr ccp get-account` to return ONLY the password without new-line -OR- figure out logic in `cmd/logon.go` to parse the password out of the full JSON response.

```shell
$ cybr logon -a ldap -b https://cyberark.joegarcia.dev -u jgarcia --non-interactive -p $(cybr ccp get-account -b https://cyberark.joegarcia.dev -i test-cli -s D-Win-SvcAccts -o "Operating System-WinDomain-10.0.4.48-Svc_CybrCLI")
2021/03/29 09:46:54 Failed to Logon to the PVWA. Failed to authenticate to the PAS REST API. Received non-200 status code '403'```